src/weights.rs

```diff
@@ -64,87 +64,93 @@ pub trait WeightInfo {
 			// Standard Error: 21_000
-			.saturating_add((Weight::from_parts(72_298_000, 0)).saturating_mul(x as u64))
-			.saturating_add(T::DbWeight::get().reads(8 as u64))
-			.saturating_add(T::DbWeight::get().reads((4 as u64).saturating_mul(x as u64)))
-			.saturating_add(T::DbWeight::get().writes(5 as u64))
-			.saturating_add(T::DbWeight::get().writes((3 as u64).saturating_mul(x as u64)))
+			.saturating_add(Weight::from_ref_time(
+				72_298_000_u64.saturating_mul(x as u64),
+			))
+			.saturating_add(T::DbWeight::get().reads(8))
+			.saturating_add(T::DbWeight::get().reads(x as u64))
+			.saturating_add(T::DbWeight::get().writes(5))
+			.saturating_add(T::DbWeight::get().writes(3 as u64))
 	}
 	fn complete_initialization() -> Weight {
-		Weight::from_parts(51_047_000, 0)
-			.saturating_add(T::DbWeight::get().reads(6 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+		Weight::from_ref_time(51_047_000)
+			.saturating_add(T::DbWeight::get().reads(6))
+			.saturating_add(T::DbWeight::get().writes(3))
 	}
 	fn set_crowdloan_allocation() -> Weight {
-		Weight::from_parts(147_000, 0)
-			.saturating_add(T::DbWeight::get().writes(1 as u64))
+		Weight::from_ref_time(147_000).saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
 	fn claim() -> Weight {
-		Weight::from_parts(101_484_000, 0)
-			.saturating_add(T::DbWeight::get().reads(11 as u64))
-			.saturating_add(T::DbWeight::get().writes(5 as u64))
+		Weight::from_ref_time(101_484_000)
+			.saturating_add(T::DbWeight::get().reads(11))
+			.saturating_add(T::DbWeight::get().writes(5))
 	}
 	fn update_reward_address() -> Weight {
-		Weight::from_parts(59_051_000, 0)
-			.saturating_add(T::DbWeight::get().reads(6 as u64))
-			.saturating_add(T::DbWeight::get().writes(4 as u64))
+		Weight::from_ref_time(59_051_000)
+			.saturating_add(T::DbWeight::get().reads(6))
+			.saturating_add(T::DbWeight::get().writes(4))
 	}
 	fn associate_native_identity() -> Weight {
-		Weight::from_parts(152_997_000, 0)
-			.saturating_add(T::DbWeight::get().reads(9 as u64))
-			.saturating_add(T::DbWeight::get().writes(7 as u64))
+		Weight::from_ref_time(152_997_000)
+			.saturating_add(T::DbWeight::get().reads(9))
+			.saturating_add(T::DbWeight::get().writes(7))
 	}
 	fn change_association_with_relay_keys(x: u32) -> Weight {
-		Weight::from_parts(0, 0)
+		Weight::from_ref_time(0)
 			// Standard Error: 7_000
-			.saturating_add((Weight::from_parts(47_373_000, 0)).saturating_mul(x as u64))
-			.saturating_add(T::DbWeight::get().reads(6 as u64))
-			.saturating_add(T::DbWeight::get().writes(4 as u64))
+			.saturating_add(Weight::from_ref_time(
+				47_373_000_u64.saturating_mul(x as u64),
+			))
+			.saturating_add(T::DbWeight::get().reads(6))
+			.saturating_add(T::DbWeight::get().writes(4))
 	}
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
 	fn initialize_reward_vec(x: u32) -> Weight {
-		Weight::from_parts(143_109_000, 0)
+		Weight::from_ref_time(143_109_000)
```

Comment:
Why did you remove from_parts and instead you have from_ref_time ? .. from_ref_time is deprecated